src/group_theory/group_action.lean

```diff
@@ -437,3 +438,21 @@ theorem smul_sub (r : α) (x y : β) : r • (x - y) = r • x - r • y :=
 by rw [sub_eq_add_neg, sub_eq_add_neg, smul_add, smul_neg]
 
 end
+
+section
+open equiv
+
+/-- This constructs the natural permutation group on `G` that is used to show Cayley's theorem-/
+def group_to_perm (G : Type*) [group G] : G →* perm G :=
+{ to_fun := λ g,
+  { to_fun := λ h, g * h,
+    inv_fun := λ h, g⁻¹ * h,
+    left_inv := fun h, by group,
+    right_inv := λ h, by group },
+  map_one' := by {ext, simp},
+  map_mul' := λ x y, by { ext, simp, group } }
+
+lemma cayley (G : Type*) [group G] : ∃ h: G →* perm G, h=h :=
+⟨ group_to_perm G, rfl ⟩
```

Comment:
This can be simplified to
```suggestion
lemma cayley (G : Type*) [group G] : ∃ h: G →* perm G, true :=
⟨ group_to_perm G, trivial ⟩
```
but perhaps would make more sense as
```suggestion
instance (G : Type*) [group G] : inhabited (G →* perm G) := group_to_perm G
```